### PR TITLE
remove doubled "hints"

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: v1.0/v1.0/envvar3.cwl [hunk 1]
```diff
@@ -6,6 +6,9 @@ outputs:
   results:
     type: File
     outputBinding: { glob: results }
+  # log:
+  #   type: File
+  #   outputBinding: { glob: log }
```
Comment (tetron) [inline v1.0/v1.0/envvar3.cwl]: Better to remove it than comment it out?

File: v1.0/v1.0/imported-hint.cwl [hunk 2]
```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env cwl-runner
 cwlVersion: v1.0
 class: CommandLineTool
-hints:
```
Comment (tetron) [inline v1.0/v1.0/imported-hint.cwl]: Is this fixed in v1.1?  Looks vaguely familiar.

File: v1.0/v1.0/envvar3.cwl (hunk 1 above)
Comment (mr-c) [reply v1.0/v1.0/envvar3.cwl]: It is nice when one is debugging this particular test.

File: v1.0/v1.0/imported-hint.cwl (hunk 2 above)
Comment (mr-c) [reply v1.0/v1.0/imported-hint.cwl]: Yes, https://github.com/common-workflow-language/cwl-v1.1/blob/83bb02793b59cb039bbf12f88812678e920f727e/tests/imported-hint.cwl

**Pull request merged**